backend/api/build_kfp_server_api_python_package.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash -e
```

Comment:
can this be put in sdk/ folder? backend/api is intend to be language agnostic. all language specific clients are currently located in the client folder.

Reply:
That's where I put it initially...